CHANGES.md

```diff
@@ -1,5 +1,27 @@
 # Changes
 
+## edge-20.7.2
+
+This edge release moves Linkerd's bundled Prometheus into an add-on. This makes
+the Linkerd Prometheus more configurable, gives it a separate upgrade lifecycle
+from the rest of the control plane, and allows users to disable the bundled
+Prometheus instance. In addition, this release includes fixes for several
```

Comment:
Prometheus is an add-on now, and the goal is indeed to be able to use a an external instance or to not use a prometheus instance at all, but these two options are not implemented yet, so you might wanna leave the "allow users to disable" out for now.

Reply:
Do you think it should be changed to "will allow" or something, or just removed?

Reply:
"will allow" sounds good to me :+1: